CMakeLists.txt

```diff
@@ -146,6 +146,8 @@ include_optional_module(ENVIRONMENT_VARIABLE WITH_OWN_QTKEYCHAIN
 
 find_package(Qt6 6.4.0 REQUIRED)
 
+find_package(Lua 5.1 EXACT)
```

Comment:
I'm surprised that `5.1` is sufficient to pin down the version but that is how it is "defined" (as `LUA_VERSION` in the header file - as `Lua 5.1`) you have to consider `LUA_RELEASE` - as `Lua 5.1.5` to get a full three part semantic version number.

OTOH If this is sufficient to get builds compiling AND linking - even without a `target_compile_definitions(mudlet PRIVATE INCLUDE_VERSIONED_LUA_HEADERS)` in the **`/src/CMakeLists.txt`** file - NOT the top level `/CMakeLists.txt`, then fair enough.

:shipit: